src/test/resources/Features/login.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Testing SWANGLABS login page
+  Background:
+    Given I open the login page
+    And the Username text box is empty
```

Comment:
De ce ai ales sa faci asta?

Reply:
Daca te referi la cele dound and-uri ma gandeam la o initializare. Adica ca si cum ai initializa o variabila cu 0 la inceput. Un fel de declarare a paginii principale.